src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <v-card flat id="relationships-panel" class="pb-4 pl-4">
-    <v-card flat rounded="sm" class="relationships-vcard px-4 mt-3 mr-9 ml-4">
+  <v-card flat id="relationships-panel">
+    <v-card flat rounded="sm" class="relationships-vcard">
```

Comment:
I just noticed this -- there shouldn't be a v-card in a v-card. Probably the second one should be a div.

Reply:
Wow, good catch @severinbeauvais  I'll fix it